integration/survey-api.md

```diff
@@ -482,14 +487,17 @@ Authorization: Bearer $survey_import
     "type": "helsenorge",
     "id": <uuid>,
     "fnr": "",
+    "status": "init"
+    "status_message": ""
     "created_date": "",
     "end_date": "",
-    "completed": false
-    "completed_date": null,
```

Comment:
@kouylekov-usit 
completed_date removed? Should I update the [schema](https://github.uio.no/IT-TSD/tsd-helsenorge-client/blob/main/schemas/helsenorge_delivery.json) too?